src/Accessors.jl

```diff
@@ -4,50 +4,35 @@ using MacroTools: isstructdef, splitstructdef, postwalk
 using InverseFunctions
 using Markdown: Markdown, @md_str, term
 
-if !isdefined(Base, :get_extension)
-    using Requires
-end
-
-
 include("setindex.jl")
 include("optics.jl")
 include("getsetall.jl")
 include("sugar.jl")
 include("functionlenses.jl")
 include("testing.jl")
 
-# always included for now
-include("../ext/AccessorsDatesExt.jl")
-include("../ext/AccessorsLinearAlgebraExt.jl")
```

Comment:
This should be kept (unless you want to demote them to weak dependencies as well in this PR).